### PR TITLE
Added docs for fossil endpoints, added quotes to paths

### PR DESCRIPTION
**Pull request by angelolz1**
For some reason, the version of the api in the YAML wasn't in their own new line as well, so I went ahead and fixed that.

I've added quotes to the paths just so that it stays consistent with the few that already have quotes.

If some of the description of the fields sounds weird, feel free to change it! My English isn't the best around, but I tried to make it as clear as possible.

If there are any errors I've made in the schema or in the paths, please let me know.

*This is my first contribution to any repo ever, so hopefully I didn't do too bad!*

### CODE REVIEW

File: static/doc.yaml [hunk 1]
```diff
@@ -3539,3 +3822,281 @@ components:
...
+          example: Spinosaurus
+          description: The name of the fossil group.
+        url:
+          type: string
+          example: 'https://nookipedia.com/wiki/Spinosaurus'
+          description: Link to the respective Nookipedia article.
+        room:
+          type: integer
+          example: 2
+          description: The floor where the fossil group can be found in the museum's fossil section.
```
Comment (KevinPayravi) [inline static/doc.yaml:3899]: Floor -> room

File: static/doc.yaml [hunk 2]
```diff
@@ -3539,3 +3822,281 @@ components:
...
```
Comment (KevinPayravi) [inline static/doc.yaml:4023]: Floor -> room

**Comment by KevinPayravi** [comment]
This is great, thanks! Just that one small change to the `room` description and we should be good to go.

**Comment by angelolz1** [comment]
Thank you for that! I've made the changes. Please let me know if there are more changes to make!

**Review by KevinPayravi — APPROVED** [review]
Thank you!

**Pull request merged**